### PR TITLE
Update @ckeditor/ckeditor5-list to v32

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ckeditor/ckeditor5/releases/tag/v32.0.0
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by fedemp** [comment]
@martin-badin Since you recently contributed to these types, can you check this PR?

File: types/ckeditor__ckeditor5-list/src/listproperties.d.ts [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import { Plugin } from '@ckeditor/ckeditor5-core';
-import ListPropertiesEditing from './listproperties/listpropertiesediting';
-import ListPropertiesUI from './listproperties/listpropertiesui';
+import ListPropertiesEditing from './listpropertiesediting';
```
Comment (martin-badin) [inline types/ckeditor__ckeditor5-list/src/listproperties.d.ts:2]: you should follow original paths. Please see https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-list/src/listproperties
The paths should be 
```typescript
import ListPropertiesEditing from './listproperties/listpropertiesediting';
import ListPropertiesUI from './listproperties/listpropertiesui';
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/ckeditor__ckeditor5-list/index.d.ts [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 export { default as List } from './src/list';
 export { default as ListEditing } from './src/listediting';
-export { default as ListProperties } from './src/listproperties';
-export { default as ListStyle } from './src/liststyle';
-export { default as ListStyleEditing } from './src/liststyleediting';
-export { default as ListStyleUI } from './src/liststyleui';
 export { default as ListUI } from './src/listui';
+export { default as ListProperties } from './src/listproperties';
+export { default as ListPropertiesEditing } from './src/listpropertiesediting';
+export { default as ListPropertiesUI } from './src/listpropertiesui';
```
Comment (martin-badin) [inline types/ckeditor__ckeditor5-list/index.d.ts:12]: These files don't exist in the repository.
https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-list/src

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/ckeditor__ckeditor5-list/src/listproperties.d.ts (hunk 1 above)
Comment (fedemp) [reply types/ckeditor__ckeditor5-list/src/listproperties.d.ts:2]: Actually we should follow the stable release (v32), not `master`.

https://github.com/ckeditor/ckeditor5/tree/v32.0.0/packages/ckeditor5-list/src

File: types/ckeditor__ckeditor5-list/index.d.ts (hunk 2 above)
Comment (fedemp) [reply types/ckeditor__ckeditor5-list/index.d.ts:12]: They do in v32.

https://github.com/ckeditor/ckeditor5/tree/v32.0.0/packages/ckeditor5-list/src/

**Review by martin-badin — APPROVED** [review]
I hope there is no issue. LGTM!

**Comment by fedemp** [comment]
> I hope there is no issue. LGTM!

It will be if somebody tries to import a file deep in the hierarchy, but not if they just do `Import List from "@ckeditor/ckeditor5-list`.

At least it's clear that this is for v32, not v33.

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**